cypress/component/AuthDialog.cy.tsx

```diff
@@ -2,22 +2,22 @@ import { GoogleOAuthProvider } from '@react-oauth/google';
 import AuthDialog from '../../src/components/AuthDialog';
 
 const props = {
-  isLoggedIn: false,
   onAuth: () => {},
+  onclose: () => {},
```

Comment:
```suggestion
  onClose: () => {},
```
consistent capitalizartion?